andela/travel_tool_front
# #160371487 Mark all notifications as read

#### What does this PR do?
* This PR contains details for marking all notifications as read

#### Description of Task to be completed?
* A user should be able to update all notifications as read when they click on the `mark all as read` button

#### How should this be manually tested?
* Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
* Switch to the repo directory - cd travel_tool_front
* Check out this branch - `git checkout ft-mark-all-notifications-as-read-#160371487`
* Install dependencies and start the server - `make start`
* Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/46)
* Navigate to `localhost:3000` in a browser
* Login with your `Andela mail`
* Click on the `Notification Icon`
* Click on the `mark all as read` button

#### Any background context you want to provide?
* N/A

#### What are the relevant pivotal tracker stories?
* [#160371487](https://www.pivotaltracker.com/story/show/160371487)

#### Screenshots (if appropriate)
* 
<img width="1433" alt="screen shot 2018-09-24 at 4 56 58 pm" src="https://user-images.githubusercontent.com/27569442/45963355-e779c500-c01a-11e8-89ea-6e6787f9f7fe.png">


#### Questions:
* N/A
